batch/batch/server/server.py

```diff
@@ -620,6 +623,10 @@ def to_dict(self):
     if 'memory' not in pod_spec.containers[0].resources.requests:
         pod_spec.containers[0].resources.requests['memory'] = '500M'
 
+    if not pod_spec.tolerations:
+        pod_spec.tolerations = []
+    pod_spec.tolerations.extend(kube.client.V1Toleration(key='preemptible', value='true'))
```

Comment:
I think we want pod_spec.tolerations to be a [V1Toleration], so I believe .append is more appropriate (or just literal `pod_spec.tolerations = [ kube.client.V1Toleration(key='preemptible', value='true') ]`

Reply:
The client may have already specified other tolerations.

Reply:
Ah, so in this case `kube.client.V1Toleration` doesn't return a single V1Toleration but a list of them?

Reply:
Sorry, you're right, I needed `append`.  Fixed.